complete_streaming_function.py

```
@@ -0,0 +1,394 @@
+                )
+                db_session.add(scene_intro_log)
+                db_session.commit()
+                print(f"[STREAM DEBUG] Saved scene intro message: order={begin_order + 2}")
+                
+                # Send metadata
+                yield f"data: {json.dumps({'done': True, 'persona_name': 'ChatOrchestrator', 'persona_id': None, 'scene_completed': False, 'next_scene_id': None, 'turn_count': 0, 'scene_intro_message': scene_intro_message, 'full_content': full_response})}\n\n"
+                return
+            
+            # Save user message
+            next_order = 1
+            last_msg = db_session.query(ConversationLog).filter(
```

Ruff (0.14.1)

Undefined name `ConversationLog`

(F821)